src/goto-programs/interpreter.cpp

```diff
@@ -551,7 +551,7 @@ exprt interpretert::get_value(
   }
   else if(real_type.id()==ID_array)
   {
-    constant_exprt result(type);
+    array_exprt result(to_array_type(real_type));
```

Comment:
Is this code being tested somewhere?

Reply:
I doubt it -- there are no tests for the interpreter, I believe.

Reply:
Not in this repository, but I believe that's happening in TG or some other private repo - @peterschrammel?